packages/library-legacy/src/connection.ts

```diff
@@ -58,6 +58,19 @@ import type {Blockhash} from './blockhash';
 import type {FeeCalculator} from './fee-calculator';
 import type {TransactionSignature} from './transaction';
 import type {CompiledInstruction} from './message';
+/** EXPERIMENTAL: Imports */
+import {Base58EncodedAddress} from '../../keys';
+import {SolanaRpcMethods} from '../../rpc-core/src';
+import {Blockhash as EXPERIMENTAL_Blockhash} from '../../rpc-core/src/blockhash';
+import {TransactionSignature as EXPERIMENTAL_TransactionSignature} from '../../rpc-core/src/transaction-signature';
+import {
+  Commitment as EXPERIMENTAL_Commitment,
+  Slot,
+  U64UnsafeBeyond2Pow53Minus1,
+} from '../../rpc-core/src/rpc-methods/common';
+import {createDefaultRpcTransport, createSolanaRpc} from '../../library/src';
+import type {IRpcTransport} from '../../rpc-transport/dist/types/transports/transport-types';
+import type {Rpc} from '../../rpc-transport/dist/types/json-rpc-types';
```

Comment:
You can add all the packages to the package.json:

```
"@solana/keys": "workspace:*",
"@solana/rpc-core": "workspace:*",
```

And then `pnpm i` and then just import them like you will in prod:

```
import {Base58EncodedAddress} from '@solana/keys';
```

Reply:
Gotcha, cool! This is what I ended up doing